### ISSUE TITLE
asynchronous multi-parallel access to qwen-turbo

### ISSUE THREAD

**Issue by cjy02044027** [XylonFu/UltimateMath#1]
Hello,

I have set the value of the Authorization keyword in config.py to "Bearer " followed by my API key from the Bailian Alibaba Cloud Large Model Service Platform. However, the response failed. May I inquire if the program encounters issues when it needs to perform asynchronous multi-parallel access to qwen-turbo? The API key has already undergone real-name authentication.

Thank you for your attention.

**Comment by XylonFu** [comment]
Thank you for your question. I have run this program again, and the only issue I encountered was: "Error: No 'content' found in response at index 34," which occurs due to the concurrency limit imposed by Alibaba. This issue can be addressed by running the program multiple times, as the internal code design can automatically handle it, or by reducing the "num" parameter to stay within the concurrency limit.

As I wrote this, I realized I might have understood what you meant. While we don't support the asynchronous API_URL provided directly by Alibaba, we have implemented a mechanism to achieve a similar asynchronous effect by using the synchronous API_URL.
